sources/procfs.go

```diff
@@ -72,6 +75,11 @@ type lustreHelpStruct struct {
 	helpText string
 }
 
+type multistatParsingStruct struct {
```

Comment:
Now that I look at this again...is it good Go form to have the word 'Struct' in the name of a struct?